### PR TITLE
Ch3 updates

### PR DESCRIPTION
**Pull request by robvet**
Combining pull request #22971.

Updates requested by Dapr team.

Updates to ms.date metadata to get added back as contributor as we lost history right before we published.

### CODE REVIEW

File: docs/architecture/dapr-for-net-developers/getting-started.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ Now, you'll configure communication between the services using Dapr [service inv
     }
     ```
 
-    The call to `AddDapr` registers the `DaprClient` class with the ASP.NET Core dependency injection system. You'll use the `DaprClient` class later on to communicate with the Dapr sidecar.
+    The call to `AddDapr` registers the `DaprClient` class with the ASP.NET Core dependency injection system. That done, you can now inject an instance of `DaprClient` into your service code to communicate with the Dapr sidecar, building blocks, and components.
```
Comment (IEvangelist) [inline docs/architecture/dapr-for-net-developers/getting-started.md:253]:
```suggestion
    The call to `AddDapr` registers the `DaprClient` class with the ASP.NET Core dependency injection system. With the client registered, you can now inject an instance of `DaprClient` into your service code to communicate with the Dapr sidecar, building blocks, and components.
```

**Review by IEvangelist — APPROVED** [review]
One minor nit, otherwise -- looks great!

**Comment by robvet** [comment]
Hey David.  Great catch.

**Review by nishanil — APPROVED** [review]

**Comment by nishanil** [comment]
@sughosneo cc for generating PDF.

**Comment by sughosneo** [comment]
> @sughosneo cc for generating PDF.

PDF has been updated now. You can download the latest. Thank you.

**Pull request merged**